ini/parser_test.go

```diff
@@ -1,15 +1,10 @@
-package test
+package ini
```

Comment:
can use testdata dir to keep separate versions of data files to test against, it's still fine to have them in code. https://eli.thegreenplace.net/2022/file-driven-testing-in-go/